### ISSUE TITLE
"Unknown" error into the pom.xml file.

### ISSUE THREAD

**Issue by gallandarakhneorg** [sarl/sarl#921]
**Describe the bug**
When compiling a Maven-based SARL project, the compiler into Eclipse (not on the command-line interface) shows up an "Unknown" error at the first line of the `pom.xml` file.

**To Reproduce**
1. Create a Maven SARL project with the following pom file:
```xml
<project xmlns="http://maven.apache.org/POM/4.0.0"
	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
	<modelVersion>4.0.0</modelVersion>
	<groupId>mygroup</groupId>
	<artifactId>myproject</artifactId>
	<version>0.1-SNAPSHOT</version>
	<properties>
		<sarl.version>0.10.0-SNAPSHOT</sarl.version>
		<janus.version>2.0.10.0-SNAPSHOT</janus.version>
		<compiler.level>1.8</compiler.level>
		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
	</properties>
	<dependencies>
		<dependency>
			<groupId>javafx</groupId>
			<artifactId>jfxrt</artifactId>
			<version>${java.version}</version>
			<scope>system</scope>
			<systemPath>${java.home}/lib/ext/jfxrt.jar</systemPath>
		</dependency>
		<dependency>
			<groupId>io.janusproject</groupId>
			<artifactId>io.janusproject.kernel</artifactId>
			<version>${janus.version}</version>
		</dependency>
		<dependency>
			<groupId>org.arakhne.afc.gis.ui</groupId>
			<artifactId>gisroadfx</artifactId>
			<version>${afc.version}</version>
		</dependency>
		<dependency>
			<groupId>org.arakhne.afc.core</groupId>
			<artifactId>mathphysics</artifactId>
			<version>${afc.version}</version>
		</dependency>
		<dependency>
			<groupId>org.arakhne.afc.core</groupId>
			<artifactId>mathstochastic</artifactId>
			<version>${afc.version}</version>
		</dependency>
		<dependency>
			<groupId>org.arakhne.afc.core</groupId>
			<artifactId>vmutils</artifactId>
			<version>${afc.version}</version>
		</dependency>
		<dependency>
			<groupId>com.dlsc.preferencesfx</groupId>
			<artifactId>preferencesfx-core</artifactId>
			<version>2.1.0</version>
		</dependency>
		<dependency>
			<groupId>org.fxmisc.richtext</groupId>
			<artifactId>richtextfx</artifactId>
			<version>0.10.1</version>
		</dependency>
		<dependency>
			<groupId>com.opencsv</groupId>
			<artifactId>opencsv</artifactId>
			<version>4.6</version>
		</dependency>
	</dependencies>

	<build>
		<plugins>
			<plugin>
				<groupId>org.apache.maven.plugins</groupId>
				<artifactId>maven-compiler-plugin</artifactId>
				<version>3.8.1</version>
				<configuration>
					<source>${compiler.level}</source>
					<target>${compiler.level}</target>
					<encoding>${project.build.sourceEncoding}</encoding>
				</configuration>
			</plugin>
			<plugin>
				<groupId>io.sarl.maven</groupId>
				<artifactId>sarl-maven-plugin</artifactId>
				<version>${sarl.version}</version>
				<extensions>true</extensions>
				<configuration>
					<source>${compiler.level}</source>
					<target>${compiler.level}</target>
					<encoding>${project.build.sourceEncoding}</encoding>
				</configuration>
			</plugin>
		</plugins>
	</build>
	<repositories>
		<repository>
			<name>Arakhne Lastest Builds</name>
			<id>org.arakhne.maven</id>
			<url>http://download.arakhne.org/arakhne/maven</url>
			<releases>
				<enabled>true</enabled>
			</releases>
			<snapshots>
				<enabled>true</enabled>
			</snapshots>
		</repository>
	</repositories>
</project>
```
2. Compile
3. See error

**Expected behavior**
No error is expected.

**Screenshots**
![Capture d’écran de 2019-06-14 14-23-54](https://user-images.githubusercontent.com/4200154/59509044-3a005780-8eb0-11e9-8c08-cfaa846e4c14.png)

**System configuration:**
- SARL version: 0.9.0 and 0.10.0
- SARL compiler:	
-  [ ] Eclipse compiler without Maven	
-  [x] Eclipse compiler with Maven	
-  [ ] Maven compiler on the command line	
-  [ ] sarlc compiler	
- Java JDK version (with the manufacturer name): 1.8.0_201
- Operating System:	
- [x] Linux 64bits	
- [ ] Linux 32bits	
- [ ] Windows 64bits	
- [ ] Windows 32bits	
- [ ] MacOS 64bits

**Comment by gallandarakhneorg** [comment]
There is incompatibilities between the version of M2E and the version 3.1.2 of the maven-jar-plugin.

Until the problem is solved by the development teams of M2E or maven-jar-plugins, you have to add in your `pom.xml` file the following line in order to solve this issue:

```xml
</project>
	<build>
		<pluginManagement>
			<plugins>
				<plugin>
					<groupId>org.apache.maven.plugins</groupId>
					<artifactId>maven-jar-plugin</artifactId>
					<version>3.1.1</version>
				</plugin>
			</plugins>
		</pluginManagement>
	</build>
</project>
```

**Comment by v2vdeveloper** [comment]
Thank you sir it is working fine...!

**Comment by vishnuvardhan-cmd** [comment]
what is the reason behind the occuring of unknown error

**Comment by gallandarakhneorg** [comment]
M2E plugin throw an error. The error is not generated by the SARL plugins but by the M2E plugins.

**Comment by DeblinaMukherjee** [comment]
Thank you so much. It was useful.

**Comment by alvarov66** [comment]
Thank you, it worked perfectly.
Notice the newer version "3.2.0" shows up the same error.

**Comment by gallandarakhneorg** [comment]
Thank you for the update. We will move to the new Eclipse 2020-06 soon (Issue #992). We will check if the new version of M2E is compatible with more recent version of `maven-jar-plugin`.

**Comment by gallandarakhneorg** [comment]
Seems to be fixed into SARL 0.12, that is based on Eclipse 2020-06. We do not need to specify the `maven-jar-plugin` anymore.
